src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -300,20 +312,26 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
 
             // Add all listable functions details to the payload
             JObject functions = new JObject();
-            string routePrefix = await WebFunctionsManager.GetRoutePrefix(hostOptions.RootScriptPath);
+
             var listableFunctions = _functionMetadataManager.GetFunctionMetadata().Where(m => !m.IsCodeless());
             var functionDetails = await WebFunctionsManager.GetFunctionMetadataResponse(listableFunctions, hostOptions, _hostNameProvider);
             result.Add("functions", new JArray(functionDetails.Select(p => JObject.FromObject(p))));
 
             // TEMP: refactor this code to properly add extensions in all scenario(#7394)
             // Add the host.json extensions to the payload
-            if (_environment.IsKubernetesManagedHosting())
+
+            // Glenna testing: Include the extensions payload for all apps
+            JToken hostJsonPayload = _configuration.Convert("AzureFunctionsJobHost");
+            JToken extensionsPayload = hostJsonPayload["extensions"];
```

Comment:
Would this result in a index invalid error? If the extensions section does not exist?

Reply:
Thanks. I tested bymyself. It looks good. In this case.  hostJsonPayload will never be null. extensionPayload will be null. I have null check just after this section. 
![image](https://user-images.githubusercontent.com/1390976/145733366-67d243fc-7910-4387-8c16-a3ac6ac38770.png)

Reply:
For the new logic, extensions section will always exists.